### PR TITLE
l10n: Add a colon to text string

### PR DESCRIPTION
**Pull request by Valdnet**
Fix the colon https://github.com/nextcloud/serverinfo/pull/333#discussion_r858374769.

@nickvergessen Many thanks for solving the problem quickly 🥇.

Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

**Comment by nickvergessen** [comment]
Can you also fix all `')); ?>:` occurrences?

File: templates/settings-admin.php [hunk 1]
```diff
@@ -266,25 +266,21 @@ class="barchart"
...
 						</p>
 						<p>
-							<?php p($l->t('Upload max size').':'); ?>
+							<?php p($l->t('Upload max size:')); ?>
 							<em id="phpUploadMaxSize"><?php p($_['php']['upload_max_filesize']); ?></em>
 						</p>
-						<p>
-							<?php p($l->t('Extensions:')); ?>
-							<em id="phpExtensions"><?php p($_['php']['extensions'] !== null ? implode(', ', $_['php']['extensions']) : $l->t('Unable to list extensions')); ?></em>
-						</p>
```
Comment (come-nc) [inline templates/settings-admin.php:287]: ?

**Review by come-nc — CHANGES_REQUESTED** [review]

**Comment by Valdnet** [comment]
> Can you also fix all `')); ?>:` occurrences?

Done. Thanks for pointing that out :wink:.

File: templates/settings-admin.php (hunk 1 above)
Comment (Valdnet) [reply templates/settings-admin.php:287]: Is there something wrong with this code?

Comment (come-nc) [reply templates/settings-admin.php:287]: Why do you remove it?

Comment (R0Wi) [reply templates/settings-admin.php:287]: Looks suspicious to me, too. Code was introduced by https://github.com/nextcloud/serverinfo/pull/367/

Comment (Valdnet) [reply templates/settings-admin.php]: Oops. I didn't notice it. Sorry, I was working with an older version of the file. I have already brought back an earlier shift 😉.

**Review by come-nc — APPROVED** [review]

**Pull request merged**